spacewalk/config/etc/httpd/conf.d/zz-spacewalk-www.conf

```diff
@@ -52,6 +52,10 @@ RewriteRule ^/WEBRPC /rhn/rpc/api
 RewriteRule ^/rhn/manager/download/dists/(.*)/main/(.*)/(Packages.*)$ /rhn/manager/download/$1/repodata/$3
 RewriteRule ^/rhn/manager/download/dists/(.*)/(InRelease|Release.*)$ /rhn/manager/download/$1/repodata/$2
 
+# Workaround for SLE11 Kiwi that unconditionally appends extra params to https urls
```

Comment:
Why only for SLE11?

Reply:
This is some feature/bug in SLES11 kiwi, probably related to old NCC. It is safer to fix it here than touching the kiwi package.
SLES12 Kiwi does not need it.